sandbox-templates/mdx-loader-next/next.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
Probably best to be in sync with: https://mdxjs.com/docs/getting-started/#nextjs

Reply:
Synced with one difference, skipped the ESM option as it is not needed in Next 12 https://nextjs.org/blog/next-12#es-modules-support-and-url-imports